bagisto/bagisto
# Breadcrumb feature added in shop account section for both themes #5158

## Issue Reference
Link: https://github.com/bagisto/bagisto/issues/5158

## Description
- Breadcrumb feature added in shop account section for both themes.
- Yielding done in default theme to avoid code duplicacy.
- Indentation, commenting and code cleanups are done.

## How To Test This?
- Run `composer install` first.
- Then publish vendors i.e. `php artisan vendor:publish --all --force`.
- Run `php artisan optimize`.
- Check the account section.
